src/main/kotlin/com/example/config/Config.kt

```diff
@@ -192,6 +204,43 @@ class KeycloakSecurityConfig(private val accountService: AccountService) : Keycl
     fun configResolver() = KeycloakConfigResolver { kcDeployment }
 }
 
+/**
+ *   Help on setting up a full broker: (scaling up websockets)
+ *      https://raymondhlee.wordpress.com/2014/04/12/set-up-a-full-broker-for-spring-4-stomp-over-websocket-messaging-using-activemq/
+ */
+
+@Configuration
+@EnableWebSocketMessageBroker
+class WebSocketConfig : AbstractWebSocketMessageBrokerConfigurer() {
+    companion object : KLogging()
+    override fun configureMessageBroker(config: MessageBrokerRegistry) {
+        //config.enableStompBrokerRelay("/topic").setRelayHost("host").setRelayPort(1000).setSystemHeartbeatReceiveInterval(12)
```

Comment:
Its intentional.  This way when we want to be able to scale less to look up and more to information to go on.  But I can remove it now that it would be in source control.